cellpack/autopack/loaders/recipe_loader.py

```diff
@@ -165,17 +165,10 @@ def _migrate_ingredient(old_ingredient):
         new_ingredient = {}
         for attribute in list(old_ingredient):
             if attribute in v1_to_v2_name_map:
+                value = old_ingredient[attribute]
                 if attribute == "Type":
```

Comment:
```suggestion
for attribute in list(old_ingredient):
            if attribute in v1_to_v2_name_map:
                if attribute == "Type":
                    value = ingredient_types_map[old_ingredient[attribute]]
                else: 
                    value = old_ingredient[attribute]
```

Reply:
I'm wondering if we could bring line168 under an else statement so the value of "Type" won't need to be assigned twice?

Reply:
yeah that's fine. I doesn't really matter either way